build.gradle.kts

```diff
@@ -12,33 +15,41 @@ repositories {
 }
 
 plugins {
-    id("org.jetbrains.intellij").version("0.7.3")
-    id("com.github.ben-manes.versions") version "0.38.0"
-    kotlin("jvm").version("1.4.32")
+    id("org.jetbrains.intellij") version "1.1.4"
+    id("com.github.ben-manes.versions") version "0.39.0"
+    kotlin("jvm") version embeddedKotlinVersion
     id("com.github.breadmoirai.github-release") version "2.2.12"
 }
 
 dependencies {
     implementation("io.gitlab.arturbosch.detekt:detekt-api:$detektVersion")
     implementation("io.gitlab.arturbosch.detekt:detekt-tooling:$detektVersion")
+
     runtimeOnly("io.gitlab.arturbosch.detekt:detekt-core:$detektVersion")
     runtimeOnly("io.gitlab.arturbosch.detekt:detekt-rules:$detektVersion")
     runtimeOnly("io.gitlab.arturbosch.detekt:detekt-formatting:$detektVersion")
+
     testImplementation("io.gitlab.arturbosch.detekt:detekt-test-utils:$detektVersion")
-    testImplementation("org.assertj:assertj-core:3.19.0")
+    testImplementation("org.assertj:assertj-core:3.20.2")
     testImplementation("org.junit.jupiter:junit-jupiter:5.7.2")
 }
 
+val jvmVersion = JavaVersion.VERSION_11
```

Comment:
Why are we bumping this?

Reply:
1. in GitHub Flow, the check went exactly in this version
2. minimum version of IDEA (2020.3.4) uses exactly this version of JVM
3. gradle-intellij-plugin only works with this version

I think that you need to build the plugin with the minimum version of IDEA and check compatibility for older versions